pkg/cmd/opts/install.go

```diff
@@ -223,25 +223,39 @@ func (o *CommonOptions) InstallGlooctl() error {
 }
 
 // InstallKustomize installs kustomize
-func (o *CommonOptions) InstallKustomize() error {
+func (o *CommonOptions) InstallKustomize(installationDir ...string) (err error) {
```

Comment:
_installationDir ...string_ - a vararg/array for the _installationDir_? I would think that if we really need to pass the installation dir a single string should do.